packages/react/src/components/Card/__stories__/Card.stories.js

```diff
@@ -32,19 +34,22 @@ export default {
         ),
         copy: text('copy', '', groupId),
         inverse: boolean('inverse', false, groupId),
-        cta: {
-          href: text('Cta href (cta.href)', 'https://example.com', groupId),
-          copy: text('Cta copy (cta.copy)', 'Card cta text', groupId),
-          icon: {
-            src: ArrowRight20,
-          },
-          iconPlacement: select(
-            'Cta icon placement (cta.iconPlacement)',
-            ['left', 'right'],
-            'right',
-            groupId
-          ),
-        },
+        cta: !DDS_CARD_WITH_PICTOGRAM
+          ? {
+              href: text('Cta href (cta.href)', 'https://example.com', groupId),
+              copy: text('Cta copy (cta.copy)', 'Card cta text', groupId),
+              icon: {
+                src: ArrowRight20,
+              },
+              iconPlacement: select(
+                'Cta icon placement (cta.iconPlacement)',
+                ['left', 'right'],
+                'right',
+                groupId
+              ),
+            }
+          : null,
+        pictogram: DDS_CARD_WITH_PICTOGRAM ? <Bee /> : null,
```

Comment:
I wonder if we should show as a knob to use pictogram instead of cta...

Reply:
Makes sense. What about a boolean knob, like: "Show pictogram (hides CTA):"?